packages/common/src/key-pair-provider.ts

```diff
@@ -0,0 +1,151 @@
+/**
+ * Provides encryption and decryption methods with a consistent API but
+ * differing mechanisms for dealing with encryption keys.
+ */
+
+import noop from 'lodash/noop';
+import AWS from 'aws-sdk';
+import forge from 'node-forge';
+
+import { deprecate } from './util';
+import { inTestMode } from './test-utils';
+
+export { KMS } from './kms';
+
+const getLocalStackHost = () => {
+  if (process.env.LOCAL_S3_HOST) {
+    return process.env.LOCAL_S3_HOST;
+  }
+
+  if (!process.env.LOCALSTACK_HOST) {
+    throw new Error('The LOCALSTACK_HOST environment variable is not set.');
+  }
+
+  return process.env.LOCALSTACK_HOST;
+};
+
+const buildS3Client = () => {
```

Comment:
So we re-implemented this here to be able to delete `common/aws.js` and not introduce a dependency on `aws-client`? If so, is this file going away soon? Because I don't love the duplicate logic

Reply:
Yes, this is going away in the next release.